src/client/extension.ts

```diff
@@ -270,6 +263,47 @@ export async function activate(
         vscode.commands.executeCommand('setContext', 'powerpages.websiteYmlExists', false);
     }
 
+    const registerPreviewShowCommand = async () => {
+        const isEnabled = isSiteRuntimePreviewEnabled();
+
+        _telemetry.sendTelemetryEvent("EnableSiteRuntimePreview", {
+            isEnabled: isEnabled.toString(),
+            websiteURL: websiteURL
+        });
+        oneDSLoggerWrapper.getLogger().traceInfo("EnableSiteRuntimePreview", {
+            isEnabled: isEnabled.toString(),
+            websiteURL: websiteURL
+        });
+
+        if (!isEnabled || websiteURL === "") {
+            // portal web view panel
+            _context.subscriptions.push(
+                vscode.commands.registerCommand(
+                    "microsoft-powerapps-portals.preview-show",
+                    () => {
+                        _telemetry.sendTelemetryEvent("StartCommand", {
+                            commandId: "microsoft-powerapps-portals.preview-show",
+                        });
+                        oneDSLoggerWrapper.getLogger().traceInfo("StartCommand", {
+                            commandId: "microsoft-powerapps-portals.preview-show"
+                        });
+                        PortalWebView.createOrShow();
+                    }
+                )
+            );
+        } else {
+            _context.subscriptions.push(
+                vscode.commands.registerCommand(
+                    "microsoft-powerapps-portals.preview-show",
```

Comment:
Should we add telemetry here also, like above command?

Reply:
added a common telemetry irrelevance of enableSiteRuntimePreview config enabled or disabled